lib/rspec/rabl/attribute_matcher.rb

```diff
@@ -11,7 +11,7 @@ def description
       end
 
       def failure_message
-        "expected #{parsed} to render #{rendered_attribute} = #{expected_value}"
+        "expected #{expected_value.inspect} in #{attribute_path}\n  got #{rendered_value.inspect}"
```

Comment:
Does RSpec have a standard for how they render expected and actual values? I think there must be a standard here that we can follow.